composer.json

```diff
@@ -3,11 +3,15 @@
     "description": "Laravel plugin that helps authenticate with the auth0 service",
     "license": "MIT",
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=7.1",
```

Comment:
I changed this because Laravel 5.6 already dropped support for PHP 7.0 and It was easier to resolve conflicts in the PHPUnit / Orechstra versions. 

I can imagine that this might be a problem. Let me know what you think about this :)

Reply:
Even if Laravel dropped support for old PHP versions, this change seems to be leaving out people that today are using it on old PHP versions. @joshcanhelp what is your opinion?